recipes/palp/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+export LD_LIBRARY_PATH=$PREFIX/lib:$LD_LIBRARY_PATH
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+
+cp -f GNUmakefile Makefile
+
+for dim in 4 5 6 11; do
```

Comment:
This build script deserves some comments :wink: (You can do that later in the feedstock.)